src/utils/src/Math.c

```diff
@@ -0,0 +1,41 @@
+#include "Include_i.h"
+
+/**
+ * Calculates power using Divide and Conquer style - O(log exponent) complexity
+ *
+ * To prevent overflows -
+ * For smaller base values (<=5), the exponent limit is upto 27
+ * For larger base values (<=100), the exponent limit is upto 9
+ * For ultra large base values (<=1000), the exponent limit is upto 6
+ */
+STATUS computePower(UINT64 base, UINT64 exponent, PUINT64 result) {
+    STATUS retStatus = STATUS_SUCCESS;
+    UINT64 _result = 1;
```

Comment:
starting variable names with _ is a break from the established pattern can you just rename it so something else?